src/wrappers/React.js

```diff
@@ -24,12 +24,19 @@ const makeReactContainer = Component => {
     }
 
     render () {
-      const { children, ...rest } = this.state
+      const {
+        children,
+        // Vue attaches an event handler, but it is missing an event name, so
+        // it ends up using an empty string. Prevent passing an empty string
+        // named prop to React.
+        '': invoker,
```

Comment:
One more thing, let's call this `_invoker` to indicate that we don't use this variable anywhere.